components/button/button-add.js

```diff
@@ -109,19 +93,103 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 	render() {
 		const text = this.text || this.localize('components.button-add.addItem');
 		const id = !this.textVisible ? this._buttonId : undefined;
+		const hoverFocusIcon = this._hasHover || this._hasFocus;
+
+		const content = this.textVisible
+			? html`<d2l-button-add-icon-text ?hover-focus-icon="${hoverFocusIcon}" text="${text}"></d2l-button-add-icon-text>`
+			: this._renderWithTextHidden(text, hoverFocusIcon);
 
 		return html`
-			<button class="d2l-label-text" id="${ifDefined(id)}">
+			<button
+				@blur="${this._onBlur}"
+				@focus="${this._onFocus}"
+				@mouseenter="${this._onMouseEnter}"
+				@mouseleave="${this._onMouseLeave}"
+				class="d2l-label-text d2l-visible-on-ancestor-target"
+				id="${ifDefined(id)}"
+				type="button">
+				<div class="line"></div>
+				${content}
 				<div class="line"></div>
-				<div class="content">
-					<d2l-icon icon="tier1:plus-default"></d2l-icon>
-					${this.textVisible
-		? html`<span>${text}</span>`
-		: html`<d2l-tooltip class="vdiff-target" offset="18" for="${this._buttonId}" for-type="label">${text}</d2l-tooltip>`}
-				</div>
 			</button>
 		`;
 	}
+
+	_onBlur() {
+		this._hasFocus = false;
+	}
+
+	_onFocus() {
+		this._hasFocus = true;
+	}
+
+	_onMouseEnter() {
+		this._hasHover = true;
+	}
+
+	_onMouseLeave() {
+		this._hasHover = false;
+	}
```

Comment:
Separated these out to cover the case of focus + hover then remove focus should still show the hover/focus icon